cmd/did-method-cli/sample/publickeys.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+  "id": "key1",
+  "type": "Ed25519VerificationKey2018",
+  "encoding": "Jwk",
+  "keyType": "Ed25519",
+  "purpose": ["verificationMethod"],
+  "value": "iZ+t8gA5DCW1P5bpVgHPIyk3QqzBOu9fohinLdkR4VE="
+ },
+ {
+  "id": "key2",
+  "type": "JwsVerificationKey2020",
+  "encoding": "Jwk",
+  "keyType": "P256",
+  "recovery": true,
+  "value": "BNS2d+yMwKbLOj+zaWrRyI7xwKyNQN1W8+gydCC2Tl/c2msgFGFl8HaKnZ6eOzcrmJen7qrPTTgCQmhXvtxfPgo="
+ },
+ {
+  "id": "key3",
+  "type": "JwsVerificationKey2020",
+  "encoding": "Jwk",
+  "keyType": "P256",
+  "update": true,
```

Comment:
Same for update key, you should probably have references to files just like it was done in fabric-cli

Reply:
done